main.go

```diff
@@ -227,6 +222,13 @@ func main() {
 					Usage:   "prints status in machine-readable format (supported formats: \"json\")",
 					Aliases: []string{"f"},
 				},
+				&cli.StringFlag{
+					Name:        "schema",
+					Usage:       "Print schema of machine-readable format (supported formats: \"json\")",
+					Aliases:     []string{"s"},
+					Required:    false,
+					DefaultText: "",
+				},
```

Comment:
Should this be a separate PR? Maybe I'm missing something but I don't get the connection between removing the hidden CLI option --server and adding this status subcommand schema.

Reply:
It should be fixed now.